### PR TITLE
Added the list to the string conversion function.

### PR DESCRIPTION
**Pull request by thakuravneet**
Added  list to string conversation function

### CODE REVIEW

**Comment by thakuravneet** [comment]
Added list to string function

File: FUNCTIONS/helper.py [hunk 1]
```diff
@@ -4634,3 +4634,15 @@ def scroll_down(self, y_axis):
         self.y_axis = y_axis
         self.y_axis = int(self.y_axis)
         self.driver.execute_script(f"window.scrollTo(0, {self.y_axis})")
+
+    def listToString(self, obj):
+        self.obj = obj
+        # initialize an empty string
+        str1 = ""
```
Comment (pritwinderss) [inline FUNCTIONS/helper.py:4641]: Add meaningful name in variables

**Review by thakuravneet — COMMENTED** [review]
Added the obj1  name for str1

**Comment by thakuravneet** [comment]
Added the meaning full name

**Pull request merged**